src/thunderbots/software/radio_communication/mrf/send_reliable_message_operation.cpp

```diff
@@ -0,0 +1,92 @@
+#include "send_reliable_message_operation.h"
+
+#include "constants.h"
+
+namespace
+{
+    std::unique_ptr<USB::BulkOutTransfer> create_reliable_message_transfer(
+        USB::DeviceHandle &device, unsigned int robot, uint8_t message_id,
+        unsigned int tries, const void *data, std::size_t length)
+    {
+        assert(robot < 8);
+        assert((1 <= tries) && (tries <= 256));
```

Comment:
Could we replace this `assert` (and all other `assert`'s) with an exception? (in this case probably `std::invalid_argument`?)

Reply:
The `assert(robot < 8)` should use the `MAX_ROBOTS_OVER_RADIO` constant in the shared constants.

Reply:
fixed

Reply:
Looks like there are still several `assert` statements?